operators/src/engine/result_descriptor.rs

```diff
@@ -143,15 +151,18 @@ impl ResultDescriptor for VectorResultDescriptor {
             data_type: self.data_type,
             spatial_reference: f(&self.spatial_reference),
             columns: self.columns.clone(),
+            ..*self
         }
     }
 }
 
 /// A `ResultDescriptor` for plot queries
-#[derive(Debug, Copy, Clone, Ord, PartialOrd, Eq, PartialEq, Serialize, Deserialize)]
+#[derive(Debug, Copy, Clone, PartialEq, Serialize, Deserialize)]
```

Comment:
because the bounding box is neither because `Coordinate2d` is neither

Reply:
What are the implications? Why is it okay to just drop it?

Reply:
idk why we derive stuff we don't need. As floats are not Eq and not Ord we cannot do this stuff.

Reply:
I'm just thinking that it might be necessary for instance to put it into a hashmap.
We could just implement an ordering for Coordinates where we put NaN last and treat all NaNs equally.
Generally, we would want to disallow NaNs there, but this is plainly uncomfortable to code with.

Reply:
yeah sure we can do that, but (1) we currently don't need it, so why implement it (YAGNI) (2) this isn't really part of this PR.

Reply:
YAGNI works for implementing new stuff but not for discussing the removal of stuff. I could say removing the equal function is also not part of this PR.

So you would say, we remove it and for, for instance, caching we will think about it later, if we need it and how we will get it back again?

Reply:
> YAGNI works for implementing new stuff

> We could just implement an ordering for Coordinates where we put NaN last and treat all NaNs equally.

this would be new stuff

Reply:
It is a super strange argument that this reflects YAGNI that you can remove existing stuff because it would require work that you need to figure out how to keep it for applying changes.